tests/slr_integration/Admin/Events/__snapshots__/Controller_Test__test_associated_events_list__Events with varying status with pagination__0.snapshot.html

```diff
@@ -35,9 +35,9 @@ <h1 class="wp-heading-inline">Associated Events for Some Layout 2</h1>
 
 	<tbody id="the-list"
 				>
-				<tr id="post-{{ID}}" class="iedit author-self level-0 post-{{ID}} type-tribe_events status-draft hentry">
-			<td class="title column-title has-row-actions column-primary page-title" data-colname="Title"><strong><span>Event with layout</span> &mdash; <span class='post-state'>Draft</span></strong>
-</td><td class='tags column-tags' data-colname="Tags"></td><td class='tickets column-tickets' data-colname="Attendees">&mdash;</td><td class='date column-date' data-colname="Date">Last Modified<br />2020/01/01 at 12:00 am</td>		</tr>
+				<tr id="post-{{ID}}" class="iedit author-self level-0 post-{{ID}} type-tribe_events status-publish hentry">
+			<td class="title column-title has-row-actions column-primary page-title" data-colname="Title"><strong><span>Event with draft status</span></strong>
+<div class="row-actions"><span class='view'><a href="http://wordpress.test/?tribe_events=event-with-draft-status" rel="bookmark" aria-label="View &#8220;Event with draft status&#8221;">View</a></span></div><button type="button" class="toggle-row"><span class="screen-reader-text">Show more details</span></button></td><td class='tags column-tags' data-colname="Tags"></td><td class='tickets column-tickets' data-colname="Attendees">&mdash;</td><td class='date column-date' data-colname="Date">Published<br />2020/01/01 at 12:00 am</td>		</tr>
```

Comment:
Something weird is going on with this test. notice the from `status-draft` to `status-publish` while the text changes to `Event with draft status`. So i think the test was aiming for the event to be in draft and not published.